CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 ## Unreleased
 
+**Features**:
+
+- Extend GPU context with data for Unreal Engine crash reports ([#3144](https://github.com/getsentry/relay/pull/3144))
+
 **Bug Fixes**:
 
 - Forward metrics in proxy mode. ([#3106](https://github.com/getsentry/relay/pull/3106))
 - Do not PII-scrub code locations by default. ([#3116](https://github.com/getsentry/relay/pull/3116))
+- Use actually used GPU instead of the primary one for Unreal Engine crash reports ([#3144](https://github.com/getsentry/relay/pull/3144))
```

Comment:
I think it's fine if we just have the `Feature` entry.

Reply:
Done